mappings/net/minecraft/item/ItemPlacementContext.mapping

```diff
@@ -1,12 +1,18 @@
 CLASS net/minecraft/class_1750 net/minecraft/item/ItemPlacementContext
 	FIELD field_7903 placementPos Lnet/minecraft/class_2338;
 	FIELD field_7904 canReplaceExisting Z
+	METHOD <init> (Lnet/minecraft/class_1657;Lnet/minecraft/class_1268;Lnet/minecraft/class_1799;Lnet/minecraft/class_3965;)V
+		ARG 1 player
+		ARG 2 hand
+		ARG 3 stack
+		ARG 4 hitResult
 	METHOD <init> (Lnet/minecraft/class_1838;)V
 		ARG 1 context
 	METHOD method_16355 offset (Lnet/minecraft/class_1750;Lnet/minecraft/class_2338;Lnet/minecraft/class_2350;)Lnet/minecraft/class_1750;
 		ARG 0 context
 		ARG 1 pos
 		ARG 2 side
+	METHOD method_32760 getPlayerLookDirectionY ()Lnet/minecraft/class_2350;
```

Comment:
`getHorizontalLookDirection`?

Reply:
I assume this means rotation around the Y-axis though?

Reply:
`Entity` has `getHorizontalFacing` with the same function- I think it makes sense to name them similarly?

Reply:
To clear up any confusion: this method returns either `Direction.UP` or `Direction.DOWN` based on whichever the player is more accurately looking towards.